src/zkey/pkey.h

```diff
@@ -92,6 +92,65 @@ struct aescipherkeytoken {
 	u8  varpart[80]; /* variable part */
 } __packed;
 
+struct ccakeytoken {
+	u8  type; /* TOKEN_TYPE_INTERNAL (0x1F) for internal key token */
+	u8  version;
+	u16 length;
+	u32 reserved1;
+	u8  privtok; /* CCA private key token: 0x20 */
+	u8  priv_tok_version;
+	u16 priv_tok_len;
+	u8  aeskw; /* secure key: AESKW */
+	u8  data_sec_hash; /* data section hash */
+	u16 reserved2;
+	u8  keyusage;
+	u8  curve_type;
+	u8  key_format; /* 0x08 = encrypted internal EC key */
+	u8  section_version;
+	u16 p_len;
+	u16 associated_data_len1;
+	u8  mkvp[8];
+	u8  opk[48]; /* Object Protection Key (OPK) */
+	u16 associated_data_len2;
+	u16 formatted_section_len;
+	u8  associated_data_version;
+	u8  key_label_len;
+	u16 associated_data_len3;
+	u16 ext_associated_data_len;
+	u8  user_definable_data_len;
+	u8 curve_type2;
+	u16 p_len2;
+	u8 key_usage_flag;
+	u8 key_format_flag;
+	u8 ad_section_version;
+	u8 reserved3[3];
+	u8 encr_d[0];
+	/* followed by encrypted (D) */
+} __packed;
+
+struct eccpubtoken {
+	u8 id; /* 0x21 = public section */
+	u8 version;
+	u16 length;
+	u32 reserved1;
+	u8 curve_type;
+	u8 reserved2;
+	u16 p_bitlen;
+	u16 q_len;
+	u8 q[0];
+	/* followed by variable length q, max 133 bytes */
+} __packed;
+
+#define EP11_STRUCT_MAGIC        0x1234
+
+/*
+ * Internal used values for the version field of the key header.
+ * Should match to the enum pkey_key_type in pkey.h.
+ */
+#define TOKVER_EP11_AES  0x03  /* EP11 AES key blob (old style) */
```

Comment:
I would add more space between the define and the value and align the values of the 3 defines.
But thats up to you....

Reply:
Done.